onnx/backend/test/case/node/__init__.py

```diff
@@ -0,0 +1,38 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+from collections import namedtuple
+import importlib
+import os
+import sys
+
+from . import base
+from .base import TestCase
+
+
+def _find_submodules():
```

Comment:
you can use pkgutil to make it look nicer, e.g.: https://github.com/caffe2/caffe2/blob/master/caffe2/python/layers/__init__.py

Reply:
wow it's much better, will use it.